perfkitbenchmarker/providers/aws/aws_disk.py

```diff
@@ -33,12 +33,49 @@
 VOLUME_EXISTS_STATUSES = frozenset(['creating', 'available', 'in-use', 'error'])
 VOLUME_DELETED_STATUSES = frozenset(['deleting', 'deleted'])
 VOLUME_KNOWN_STATUSES = VOLUME_EXISTS_STATUSES | VOLUME_DELETED_STATUSES
+
+STANDARD = 'standard'
+GP2 = 'gp2'
+IO1 = 'io1'
+
 DISK_TYPE = {
-    disk.STANDARD: 'standard',
-    disk.REMOTE_SSD: 'gp2',
-    disk.PIOPS: 'io1'
+    # no need to translate 'standard' because the old and new names
```

Comment:
@stfeng I don't think it is possible for a benchmark to create a disk without specifying scratch_disk_type. However, I just read through the configs for all of the benchmarks, and I am *sure* that none of them actually do it. :-)

I added disk.STANDARD to the mapping. That raised another issue, which is that since STANDARD is now in the mapping, the translation machinery will warn the user to switch names. I added a check in `WarnAndTranslateDiskTypes` to prevent that. I tested that using `--scratch_disk_type=standard` produces a warning about the flag name, but not about the disk type name.